internal/services/user.go

```diff
@@ -108,6 +108,11 @@ func (s *tokenManager) GenerateTokens(ctx context.Context, login, password strin
 	if err != nil {
 		return "", "", true, err
 	}
+
+	if user.IsDeleted {
```

Comment:
Can you please check the Refresh method to see if it generates a new access token as well? We might need to block refreshing tokens as well.